packages/xrpl/integrationTestSequencer.js

```diff
@@ -0,0 +1,33 @@
+const Sequencer = require('@jest/test-sequencer').default
+
+class CustomSequencer extends Sequencer {
+  sort(tests) {
+    // Test structure information
+    // https://github.com/facebook/jest/blob/6b8b1404a1d9254e7d5d90a8934087a9c9899dab/packages/jest-runner/src/types.ts#L17-L21
+    const copyTests = Array.from(tests)
+    const sorted = copyTests.sort((testA, testB) => {
+      // We run escrow tests first because they rely on the ledger number via ledger_accept
+      if (testA.path.includes('escrow') && testA.path.includes('escrow')) {
```

Comment:
These are checking the same thing twice
```suggestion
      if (testA.path.includes('escrow') && testB.path.includes('escrow')) {
```